packages/core/pluggableElementTypes/RpcMethodType.ts

```diff
@@ -22,18 +27,61 @@ export default abstract class RpcMethodType extends PluggableElementBase {
 
   async serializeArguments(args: {}, _rpcDriverClassName: string): Promise<{}> {
     const blobMap = getBlobMap()
-    return { ...args, blobMap }
+    const authenticationInfoMap = {}
+
+    if (
+      args.hasOwnProperty('adapterConfig') &&
+      searchOrReplaceInArgs(args, 'internetAccountId')
+    ) {
+      return this.serializeAuthArguments(args, blobMap, authenticationInfoMap)
+    }
+    return { ...args, blobMap, authenticationInfoMap }
+  }
+
+  async serializeAuthArguments(
+    args: {},
+    blobMap: { [key: string]: File },
+    authenticationInfoMap: Record<string, string>,
+  ) {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const rootModel: any = this.pluginManager.rootModel
+    this.pluginManager.rootModel.internetAccounts.find(account => {
+      return account.internetAccountId === 'specificId'
+    })
```

Comment:
I think you accidentally committed this?

Reply:
good catch